mockall/tests/automock_nonsend.rs

```diff
@@ -19,3 +19,14 @@ fn returning_st() {
     let x = Rc::new(42u32);
     assert_eq!(43, *mock.foo(x).as_ref());
 }
+
```

Comment:
While you're here, could you also add tests for `returning_st` and `withf_st` on static methods?